life360/life360_app.groovy

```diff
@@ -562,3 +581,41 @@ void captureCookies(response) {
     }
 }
 
+void dynamicPolling() {
+
+    // if (logEnable) log.trace("dynamicPolling - INFO: dynamicPolling:${dynamicPolling} || memberInTransit:${state.memberInTransit} || dynamicPollingActive:${state.dynamicPollingActive}")
+
+    state.memberInTransit = false
+
+    state.members.each { member ->
+    
+        // FOR TESTING DYNAMIC POLLING
+/*
+        if (member["firstName"] == "YOURNAME") {
+            state.memberInTransit = true
+        }
+*/
+
+        if (member["location"].inTransit.toInteger() == 1 && member["location"].speed.toInteger() > 1) {
```

Comment:
I'm getting a NPE here (inTransit)

```
java.lang.NullPointerException: Cannot get property 'inTransit' on null object on line 599 (method handleTimerFired)
```

Reply:
I pushed NPE checks to get it running; I'll check the logs after it's been running a while

Reply:
Hey!

That's interesting... I haven't noticed that here/  I just double checked on my hub and don't see that.  I've been running errands most of the afternoon and my daughter has been out 'n' about as well, so there's a lot of shifting in and out of dynamic for there to have been an error going into that function.  I'll keep an eye on it though.

Thanks for giving it a shot!